src/components/CourseCard/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { PropTypes } from 'react'
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
 import PlayButton from '../Button/PlayButton'
-import Card from './'
+import Card from '../../Card'
```

Comment:
Since `Card` isn't actually used in the API - it's only used in specific card type components, I moved Card into the shared folder up a level, and removed it from the API (src/index.js exports) and exported the specific card type components explicitly.